packages/react-atomic/src/App.tsx

```diff
@@ -0,0 +1,60 @@
+import {Engine, HeadlessEngine, searchAppReducers} from '@coveo/headless';
+import React from 'react';
+import './App.css';
+
+class App extends React.Component {
+  private engine: Engine;
+
+  constructor(props: any) {
+    super(props);
+    this.engine = new HeadlessEngine({
+      configuration: HeadlessEngine.getSampleConfiguration(),
+      reducers: searchAppReducers,
+    });
```

Comment:
Is it necessary to create and set the engine? Why not use the `sample` prop on the `atomic-search-interface` component, and place all components under it?

I feel this is simpler, and removes the need for the `@coveo/headless` dependency?